src/app/bav/steps.js

```diff
@@ -0,0 +1,17 @@
+const nameEntry = require("./controllers/nameEntry");
+const root = require("./controllers/root");
+
+module.exports = {
+  "/": {
+    resetJourney: true,
+    reset: true,
+    entryPoint: true,
+    skip: true,
+    controller: root,
+    next: "enter-name-photo-id",
+  },
+  "/enter-name-photo-id": {
```

Comment:
it won't be photo ID in this journey, maybe we just leave it as `/enter-name`?

I would also love to use constants for these paths, what do you think?

Reply:
Aware this is just a dummy screen so happy to leave as is but FYI BAV won't be collecting names

Reply:
Added constants for paths :)